Streamline APM Product Overview (#22104)
DataDog/documentation: content/en/tracing/_index.md

</br>

## Overview

Datadog Application Performance Monitoring (APM) provides deep visibility into your applications, enabling you to identify performance bottlenecks, troubleshoot issues, and optimize your services. With distributed tracing, out-of-the-box dashboards, and seamless correlation with other telemetry data, Datadog APM helps ensure the best possible performance and user experience for your applications.

For an introduction to terminology used in Datadog APM, see [APM Terms and Concepts][1].

## Getting started

The simplest way to start with Datadog APM is with Single Step Instrumentation. This approach installs the Datadog Agent and instruments your application in one step, with no additional configuration steps required. To learn more, read [Single Step Instrumentation][27].

For setups that require more customization, Datadog supports custom instrumentation with Datadog tracing libraries. To learn more, read [Application Instrumentation][2].

## Use cases

Discover some ways Datadog APM can help support your use cases:

| You want to...| How Datadog APM can help |
| ----------- | ----------- |

> | Understand how requests flow through your system. | [Trace Explorer][21] visualizes and analyzes end-to-end traces across distributed services. |

Reviewer: ```suggestion
| Understand how requests flow through your system. | Use the [Trace Explorer][21] to query and visualize end-to-end traces across distributed services. |
```

> | Monitor service health and performance of individual services. | The [service page][26] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |

Reviewer: ```suggestion
| Monitor service health and performance of individual services. | The [service][26] and [resource pages][28] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |
```

> | Monitor service health and performance of individual services. | The [service][26] and [resource pages][28] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |

Reviewer 2: ```suggestion
| Monitor service health and performance of individual services. | The [service][26] and [resource pages][28] help you assess service health by analyzing performance metrics, tracking deployments, and identifying problematic resources. |
```

> | Correlate traces with DBM, RUM, logs, synthetics, and profiles. | [Correlate APM Data with Other Telemetry][20] to give more context to your data for more comprehensive analysis. |

Reviewer 2: ```suggestion
| Correlate traces with DBM, RUM, logs, synthetics, and profiles. | [Correlate APM Data with Other Telemetry][20] to give context to your data for more comprehensive analysis. |
```

> | Control how data flows into Datadog. | [Ingestion Controls][6] adjust service-level ingestion configuration and sampling rates. [Retention filters][7] controls which spans are retained for 15 days. |

Reviewer: ```suggestion
| Control how data flows into Datadog. | Use [Ingestion Controls][6] to adjust ingestion configuration and sampling rates by service and resource. Use [Retention filters][7] to choose which spans to retain for 15 days. |
```

[16]: /profiler/
[17]: /tracing/trace_collection/automatic_instrumentation/
[18]: /tracing/trace_collection/custom_instrumentation/
[19]: /tracing/metrics/
[20]: /tracing/other_telemetry/
[21]: /tracing/trace_explorer/
[22]: /tracing/trace_collection/automatic_instrumentation/single-step-apm/
[23]: /agent/
[24]: /tracing/metrics/metrics_namespace/
[25]: /tracing/metrics/runtime_metrics/
[26]: /tracing/services/service_page/

> [27]: /tracing/trace_collection/single-step-apm/

Reviewer: ```suggestion
[27]: /tracing/trace_collection/single-step-apm/
[28]: /tracing/services/resource_page/
```
